specification/cognitiveservices/OpenAI.Inference/models/completions.create.tsp

```diff
@@ -159,6 +161,14 @@ model Completions {
   @projectedName("csharp", "InternalCreatedSecondsAfterUnixEpoch")
   created: int32;
 
+  @doc("""
+  Content filtering results for zero or more prompts in the request. In a streaming request, 
+  results for different prompts may arrive at different times or in different orders.
+  """)
+  @added(ServiceApiVersions.v2023_06_01_Preview)
```

Comment:
when/where do I need the `added` annotations? I don't see them anywhere else

Reply:
For `operations` and `models`. Now that Travis's PR is merged, you can probably rebase (or just look at `main`) for some examples. Check out `routes.tsp` and `models/images.tsp`